.eslintrc

```
@@ -56,6 +56,17 @@
     "jsx-a11y/click-events-have-key-events": "off",
     "jsx-a11y/no-noninteractive-element-interactions": "off",
     "jsx-a11y/no-noninteractive-element-to-interactive-role": "off",
+    "no-restricted-imports": [
+      "error",
+      {
+        "paths": [
+          {
+            "name": "test-utilities/legacy",
+            "message": "Please import the modern testing library from test-utilities instead"
```

```suggestion
            "message": "Please import the modern testing library from 'test-utilities' instead"
```

A petty request for quote marks, but other than that. Thanks for looking into this!